cmd/cloud/main.go

```diff
@@ -30,7 +30,7 @@ func init() {
 	_ = rootCmd.MarkFlagRequired("database")
 
 	rootCmd.AddCommand(newCmdServer())
-	rootCmd.AddCommand(clusterCmd)
+	rootCmd.AddCommand(newCmdCluster())
```

Comment:
Using the `new` prefix on these commands is okay for now, but once we complete all the refactors I would like to change the names back as I believe it may lead to confusion.

Reply:
I have rename these functions accordingly.